src/js/components/dashboard/summary/DashboardSummaries.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+  * DashboardSummaries.jsx
+  * Created by Daniel Boos 11/12/19
+  */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import DashboardSummary from './DashboardSummary';
+
+const propTypes = {
+    appliedFilters: PropTypes.object.isRequired,
+    results: PropTypes.array
+};
+
+function getFileLabel(selectedFile) {
+    let file = '';
+    if (selectedFile.substring(0, 5) === 'cross') {
+        const sourceFile = selectedFile.charAt(6);
+        const targetFile = selectedFile.substring(7);
+        file = `Cross: ${sourceFile}/${targetFile}`;
+    }
+    else {
+        file = `File ${selectedFile}`;
+    }
+    return file;
+}
+
+export default class DashboardSummaries extends React.Component {
+    render() {
+        // TODO: May need to update this when there are multiple agencies
+        const file = getFileLabel(this.props.appliedFilters.filters.file);
+        const summaries = [];
+        let key = 0;
```

Comment:
Could we use the agency id as the key (if the idea is that there will be multiple summaries for multiple agencies)?